marketing_emails/email_kickoff_no_keynote.md

```diff
@@ -0,0 +1,30 @@
+*We are back!*
+
+PyTN is returning February 10th and 11th 2018.  Bill Israel, PyTN Chairman, and his team are extending Jason Myers' success of previous years.
+
+PyTN's goal is to have an open, accepting environment to discuss Python, the programming language.  Bill is nailing down the keynotes which we will announce soon.  Nonetheless, we want you to be a part of this experience and to be a part of achieving this goal.  First thing to do is put PyTN 2018 on your calendar.  Second thing is to buy your ticket.  Third thing is to evangelize - twitter handle: @PyTennessee, #pytn, #pytn2018
```

Comment:
This is good. I'd like to avoid the word "evangelize", though...religious, pushy overtones there. Maybe something like: "Third thing is to get the word out: ..."